test/units/modules/storage/netapp/test_na_ontap_ucadapter.py

```diff
@@ -0,0 +1,159 @@
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+''' unit tests ONTAP Ansible module: na_ontap_ucadapter '''
+
+from __future__ import print_function
+import json
+import pytest
+
+from units.compat import unittest
+from units.compat.mock import patch, Mock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+
+from ansible.modules.storage.netapp.na_ontap_ucadapter \
+    import NetAppOntapadapter as my_module  # module under test
+
+if not netapp_utils.has_netapp_lib():
+    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```

Comment:
I checked on this, and I believe the correct syntax is

pytestmark = pytest.mark.skip('skipping as missing required netapp_lib')

Otherwise, I'm seeing some error like
Using pytest.skip outside of a test is not allowed. To decorate a test function, use the @pytest.mark.skip or @pytest.mark.skipif decorators instead, and to skip a module use `pytestmark = pytest.mark.{skip,skipif}.

Can somebody from Red Hat comment?

Reply:
I consulted the [pytest docs](https://docs.pytest.org/en/latest/skipping.html) and agree with @lonico .  Since this is outside of a function, the correct syntax seems to be:

`pytestmark = pytest.mark.skip('skipping as missing required netapp_lib')`